packages/shared/src/utils/url/url.js

```diff
@@ -143,7 +143,8 @@ export const setUrlLanguage = lang => {
 export const getDerivComLink = (path = '') => {
     const host = 'https://deriv.com';
     const lang = default_language?.toLowerCase();
-    const link_lang = lang === 'en' ? '' : `/${lang}`;
+    let link_lang = '';
+    if (lang && lang !== 'en') link_lang = `/${lang}`;
```

Comment:
Can I know what was the reason for this change? to avoid a trailing slash?

Reply:
It had a bug which in case of `default_language` being undefined, `lang` turned to be undefined as string.

Reply:
Consider this
```suggestion
    const lang = default_language?.toLowerCase() ?? '';
```